R/add_transition_risk_NA_share.R

```diff
@@ -0,0 +1,98 @@
+add_transition_risk_NA_share <- function(data) {
```

Comment:
By now this seems like a common pattern. Maybe we should extract it in something like `f(data, product = x, company = y)`, where `x` and `y` are functions to apply at product and company level respectively.


```r
f <- function(data, product, company) {
  product <- x(unnest_product(data))
  company <- y(unnest_company(data))
  tilt_profile(nest_levels(product, company))
}
```

Not a priority though. Just a thought.